constants.go

```diff
@@ -92,3 +92,6 @@ func init() {
 		ServiceName = "yggdrasil"
 	}
 }
+
+const smallIndent = " "
+const mediumIndent = "  "
```

Comment:
I kinda feel like these belong in `interactive.go`. They primarily exist to
support the interactive output.

Reply:
@subpop You are right. I updated the PR.